api/GeolocationCoordinates.json

```diff
@@ -72,14 +72,24 @@
               "version_removed": "14"
             }
           ],
-          "safari": {
-            "alternative_name": "Coordinates",
-            "version_added": "5"
-          },
-          "safari_ios": {
-            "alternative_name": "Coordinates",
-            "version_added": "4.2"
-          },
+          "safari": [
+            {
+              "version_added": "13.1"
+            },
+            {
+              "alternative_name": "Coordinates",
+              "version_added": "5"
```

Comment:
This matches api.GeolocationPosition.coords, good!

But should there also be a `version_removed`?

Reply:
Added based upon testing the alt. name's existence in Safari 13.1!